docs/getting-started.rst

```diff
@@ -4,101 +4,244 @@
+Users can search protein sequences that contain specific Pfam entries in a particular arrangement by clicking **Search + By Domain architecture** in the InterPro website menu. Pfam entries that the proteins should or should not contain can be included or excluded from the domain architecture. The **Order of domain matters** option offers the possibility to arrange the domains in a particular order. The **Exact match** option fine tunes the search to find only proteins containing the selected domains (no extra domain in the proteins). Domains can be selected by entering a domain name or Pfam accession.
+
+.. figure:: images/domarch.png
+    :alt: Selecting **Search + By Domain architecture** 
+    :width: 700
+    :align: left
 
-Pfam pre-calculates the domain compositions and architectures `UniprotKB reference proteomes <https://www.uniprot.org/help/reference_proteome>`_. To see the list of proteomes, click on the 'browse' link at the top of the Pfam website, and click on a letter of the alphabet in the 'proteomes' section. By clicking on a particular organism, you will be be able to view the proteome page for that organism. From here you can view the domain organisation and the domain composition for that proteome.
+    Select **Search + By Domain architecture** in the IntePro menu, enter the desired Pfam entries and select/unselect the relevant options.
 
-The taxonomy query allows quick identification of families/domains which are present in one species but are absent from another. It can also be used to find families/domains that are unique to a particular species (note this can be very slow).
 
-Finding proteins with a specific set of domain combinations ('architectures')
-=============================================================================
 
-For a detailed study of domain architectures you can use `PfamAlyzer <http://pfam.xfam.org/search?tab=searchDomainBlock>`_. PfamAlyzer allows you to find proteins which contain a specific combination of domains and to specify particular species and the evolutionary distances allowed between domains.
+Pfam entries creation and annotation
+====================================
 
-Wikipedia annotation
-====================
+For each Pfam entry, the HMM model is run against the protein sequences belonging to the UniProt Reference Proteomes. Subsequently, Pfam curators set a statistical cut-off, known as a gathering threshold (GA) for an entry. Sequences failing to make a statistical match above this threshold are not reported as hits. The threshold is usually conservative, so that no known false positives are included in the entry. The Pfam model is then run against the whole UniProtKB database before every realease and these are the matches shown on the website.
+
+Different Pfam entries have annotations providing diverse amounts of information. Many of them have a description created by Pfam curators. Anyone can contribute to this annotation by contacting directly the curators.
```

Comment:
Anyone can contribute to this annotation by contacting directly the curators through the **Add your annotation** toolbox located on the right-hand side of the **Overview** tab.